app/backend/src/tests/test_visible_users.py

```diff
@@ -0,0 +1,17 @@
+from couchers.db import get_user_by_field
+from couchers.models import User
+from tests.test_fixtures import db, generate_user, session_scope
+
+
+def test_hidden_users(db):
+    user1, token1 = generate_user()
+    user2, token2 = generate_user()
+    user3, token3 = generate_user(is_deleted=True)
+    user4, token4 = generate_user(accepted_tos=0)
+
+    with session_scope() as session:
+        user2 = get_user_by_field(session, user2.username)
+        user2.is_banned = True
```

Comment:
You can just set this on line 8?

Reply:
'generate_user(is_banned=True)' throws errors, since you can't have a user who is both banned and active. See point 5 in my original comment in this thread.

Reply:
OK